src/screens/Sidebar/CannyBoard/component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { View, Text, WebView, ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+import { kitsuConfig } from 'kitsu/config/env/';
+import { commonStyles } from 'kitsu/common/styles';
+import { styles } from './styles';
+
+class Board extends React.Component {
+  static navigationOptions = ({ navigation }) => ({
+    title: navigation.state.params.title,
+  });
+
+  state = {
+    token: null,
+    loading: true,
+  }
+
+  componentDidMount() {
+    this.getCannySsoToken();
+  }
+
+  getCannySsoToken = async () => {
```

Comment:
@edencakir I think there's no need for `async` if `await` is not being used in the function.

Reply:
@BilalBudhani Ah, forgot to remove that after switching to `fetch`. Corrected

Reply:
Shouldn't we be using `await` here though?

Reply:
I don't think it's necessary.